js/runner_test.go

```diff
@@ -2312,7 +2312,7 @@ type multiFileTestCase struct {
 
 func runMultiFileTestCase(t *testing.T, tc multiFileTestCase, tb *httpmultibin.HTTPMultiBin) {
 	t.Helper()
-	runner, err := getSimpleRunner(t, tc.cwd+"/script.js", tc.script, tc.rtOpts, tc.fses)
+	runner, err := getSimpleRunner(t, tc.cwd+"script.js", tc.script, tc.rtOpts, tc.fses)
```

Comment:
I saw why this was done in db8dab64, but [not all `tc.cwd`s have a trailing slash](https://github.com/grafana/k6/blob/1993ac1b6d47d3504631648d3cf02aebcd5adfbf/js/runner_test.go#L2441-L2443). So in those cases this will evaluate to, e.g. `/whateverscript.js` or `/pathscript.js`. It apparently has no effect on the tests, as they still pass, so I'm not sure if we should do anything about this, just wanted to point it out.

Maybe change all `tc.cwd` to have a trailing slash?

Reply:
You're right here, I've answered Mihail in another tread https://github.com/grafana/k6/pull/2928#discussion_r1120543725 Hope that works :crossed_fingers: